### DISCUSSION TITLE
Setting for Filtered view in file manager.

### DISCUSSION THREAD

**Discussion by antoniopaolini** [gsantner/markor#2115] [Ideas] [answered]
Hello, 
i'm syncing my todo dir with syncthing and then I have a lot of file that I don't want to see.   
There's some `.stfolder`, a `.DS_Store`,  the Markor's `.app` directory, etc.

I wonder if we can set an option to hide dot files from the Markor's  File view.

Thanks.

**Comment by gsantner** [comment, marked as answer]
There is a checkbox dotFiles in the top menu.

**Reply by antoniopaolini** [reply]
I newer saw it. Really Sorry.
Thanks.
